### ISSUE TITLE
toastr stylesheet conflicts with my apps bootstrap stylesheet

### ISSUE THREAD

**Issue by proehlen** [CodeSeven/toastr#345]
I'm using your package extensively in several meteor apps (thanks!).  But I'm having a bit of a problem in that I've lost track of the number of times I'm finding the toastr bootstrap stylesheet styles being applied to other elements from other modules in my project.   Is there way to get around this?  Can the toastr style names be prefixed with a namespace like "toastr-"?

**Comment by TimFerrell** [comment]
I'm seeing that all of the styles are already prefixed with `toast-`.

What exact issues are you seeing?

**Comment by proehlen** [comment]
My apologies.  In setting up / writing up an example, I realized I was mistaken.

Meteor combines all of the CSS style sheets in your meteor app into one file.  Whenever I saw that one of my styles was being overwritten, I'd scroll to the top of the CSS file and see a "toastr" notice. I think this was because one of the very first packages I added to my project was "toastr".

Sorry to have wasted your time!
